### PR TITLE
Add benefits of using packit to faq

### LINKED ISSUE #794: Document aditional benefits of Packit compared to usage of external services directly
Packit delegates most of the work to external services and here and there, we are asked to provide the benefits of Packit compared to using those services directly.

Let's provide a summary of these benefits on our documentation page.

This info needs to be linkable and can be either put on the homepage or maybe in the FAQs. (Other suggestions welcome.) Alternatively, we can provide this info for each job separately, but most probably, the benefits are generic.

### PR DESCRIPTION
**Pull request by kapr200**
Adds the Documentation about aditional benefits of Packit compared to usage of external services directly.

Fixes #794

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -157,3 +157,9 @@ You can read more about this transition [here](/posts/copr-srpms).
 ## A command failed in packit-service: how do I reproduce it locally?
 
 Please read [this document on how to reproduce locally](reproduce-locally).
+
+## Unlocking Efficiency: The Advantages of Using Packit for Seamless Service Integration
+
+- Packit uses various services Testing Farm, Copr, Koji, Bodhi…
+- These services can be used without Packit, but Packit intergrates their continuity (the build in Copr is created automatically when a new version is released).
```
Comment (majamassarini) [inline docs/faq.md:164]:
```suggestion
- These services can be used without Packit, but Packit integrates them (the build in Copr is created automatically when a new version is released).
```
My english is bad, but the sentence above sounds to me better in the way I propose, but feel free to ignore my suggestion (just apply the typo fix :-))

**Review by majamassarini — APPROVED** [review]

File: docs/faq.md [hunk 2]
```diff
@@ -157,3 +157,9 @@ You can read more about this transition [here](/posts/copr-srpms).
 ## A command failed in packit-service: how do I reproduce it locally?
 
 Please read [this document on how to reproduce locally](reproduce-locally).
+
+## Unlocking Efficiency: The Advantages of Using Packit for Seamless Service Integration
```
Comment (lachmanfrantisek) [inline docs/faq.md:161]: Since this is a `faq` part, I would try to convert this into a question.

File: docs/faq.md (hunk 1 above)
Comment (mfocko) [reply docs/faq.md:164]: Sounds better to me :eyes:

File: docs/faq.md (hunk 2 above)
Comment (lachmanfrantisek) [reply docs/faq.md:161]: Maybe only me, but `Seamless Service Integration` seems a bit too buzzwordy to me...;) I think we don't need to create much hype in Frequently Asked Questions..;)

File: docs/faq.md (hunk 1 above)
Comment (lachmanfrantisek) [reply docs/faq.md:164]: I would also maybe explicitly emphasise that in some workflows (mainly Fedora maintenance automation), we are chaining the tasks run on these services so people do not need to be busy waiting.

**Review by lachmanfrantisek — COMMENTED** [review]
Thank you! This is important and not easy to write in a few simple sentences I must say.

File: docs/faq.md [hunk 3]
```diff
@@ -157,3 +157,9 @@ You can read more about this transition [here](/posts/copr-srpms).
...
+
+- Packit uses various services Testing Farm, Copr, Koji, Bodhi…
+- These services can be used without Packit, but Packit intergrates their continuity (the build in Copr is created automatically when a new version is released).
+- The user does not have to wait for the results and just interprets them.
```
Comment (lbarcziova) [inline docs/faq.md:165]: here I would emphasize also that the results can be accessed from one place (though indirectly), the forge/dashboard

File: docs/faq.md (hunk 1 above)
Comment (lbarcziova) [reply docs/faq.md:164]: +1, I would in general extend this a little bit more, e.g. with how we take care of passing the built NVRs of RPM builds to Testing Farm

**Review by mfocko — APPROVED** [review]

File: docs/faq.md [hunk 4]
```diff
@@ -157,3 +157,9 @@ You can read more about this transition [here](/posts/copr-srpms).
 ## A command failed in packit-service: how do I reproduce it locally?
 
 Please read [this document on how to reproduce locally](reproduce-locally).
+
+## Why you should use Packit instead of external devices directly?
+
+- Packit uses various services Testing Farm, Copr, Koji, Bodhi…
+- These services can be used without Packit, but Packit integrates them (the build in Copr is created automatically when a new version is released).
```
Comment (mfocko) [inline docs/faq.md:164]:
```suggestion
- These services can be used without Packit, but Packit integrates them in a
  more user-friendly way reducing the manual labor that would be required from
  the contributor or maintainer. For example:
  - the build in Copr is created automatically when a new version is released
  - RPMs built by Copr are specifically passed to the Testing Farm for testing
  - RPM builds are verified against multiple releases (and also different OSs)
```

@lbarcziova WDYT?

**Review by mfocko — APPROVED** [review]

File: docs/faq.md (hunk 4 above)
Comment (lbarcziova) [reply docs/faq.md]: LGTM, much better, thanks!

**Pull request merged**